### PR TITLE
Consent Modal (DEV-642)

### PR DESCRIPTION
**Pull request by davelindqvist**
[DEV-642](https://betterangels.atlassian.net/browse/DEV-642?atlOrigin=eyJpIjoiNTNhNzg0ZjBlYTI1NGI5M2E1NTAxMGFiMTkwM2M2YzQiLCJwIjoiaiJ9)

NOTE: The iPad styling is quite off. Unsure how to approach this, but at least it covers up the clients.
image

[DEV-642]: https://betterangels.atlassian.net/browse/DEV-642?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/betterangels/src/app/(tabs)/_layout.tsx [hunk 1]
```diff
@@ -249,6 +249,7 @@ export default function TabLayout() {
       <MainPlusModal closeModal={closeModal} isModalVisible={isModalVisible} />
       <ConsentModal
         user={user}
+        height={'90%'}
```
Comment (Davit-BetterAngels) [inline apps/betterangels/src/app/(tabs)/_layout.tsx:252]: I would rather do only offset from top or offset + what ui needs. 
90% will look different from device to device. 

const insets = useSafeAreaInsets();
const topOffset = insets.top;

**Comment by shiozakiyuko** [comment]
Hey @davelindqvist and @Davit-BetterAngels 
I can create a ticket for tablet issue since we are not dealing with this currently so you guys can move on.
I think we will plan to deal with it in Oct.

**Comment by Davit-BetterAngels** [comment]
border radius of the modal is wrong. not 24.

**Comment by Davit-BetterAngels** [comment]
Screenshot 2024-09-05 at 2 41 14 PM
broken on small screen

File: libs/expo/betterangels/src/lib/ui-components/ConsentModal.tsx [hunk 2]
```diff
@@ -267,10 +248,11 @@ const styles = StyleSheet.create({
   checkbox: {
     flexDirection: 'row',
   },
-  buttons: {
-    flex: 1,
-    position: 'relative',
-    bottom: -40,
+  buttonsContainer: {
+    position: 'absolute',
```
Comment (Davit-BetterAngels) [inline libs/expo/betterangels/src/lib/ui-components/ConsentModal.tsx:252]: bad idea to put under absolute

File: libs/expo/betterangels/src/lib/ui-components/ConsentModal.tsx [hunk 3]
```diff
@@ -188,44 +189,24 @@ export default function ConsentModal(props: IMainModalProps) {
             }}
           />
           <Image
+            style={{
+              marginTop: '10%',
```
Comment (Davit-BetterAngels) [inline libs/expo/betterangels/src/lib/ui-components/ConsentModal.tsx:193]: we should not differ position of the image based on the device from top unless it is required.

**Review by Davit-BetterAngels — CHANGES_REQUESTED** [review]

**Review by Davit-BetterAngels — APPROVED** [review]

**Comment by shiozakiyuko** [comment]
It works as expected. Thanks so much!

One question. Out of curiosity, is there any way to make the image less fuzzy? 
I thought it's my vision but I asked my team and they also see it a little fuzzy.

Let me know if you can make it better. Thanks!

**Comment by shiozakiyuko** [comment]
Hi @davelindqvist ,

Please use this SVG file for the image.

Figma link for [the Sign Up frame](https://www.figma.com/design/waIeX8dknHKedTrVxCQAAG/HIFI---1st-MVPs?node-id=14537-86956&t=lhFrx8cXThijlExz-4)

Figma link for the [SVG file](https://www.figma.com/design/waIeX8dknHKedTrVxCQAAG/HIFI---1st-MVPs?node-id=15483-306890&t=lhFrx8cXThijlExz-4)

Thanks so much!

**Comment by shiozakiyuko** [comment]
Hi @davelindqvist 
I just tested this and looks wonderful! Thanks so much!

**Comment by shiozakiyuko** [comment]
Hey @davelindqvist ,

I reviewed on both iOS and Android. They all look great! Thanks so much!

**Review by shiozakiyuko — APPROVED** [review]

**Pull request merged**